_v1_3_1-docs/quick-start.md

```diff
@@ -25,14 +25,9 @@ At the moment, we do not offer evaluation licenses and trials. If you want to ev
 
 ## 2. Install
 
-Check out the <a href="{{ site.baseurl }}/v1_0_0-docs/installation">Installation</a> page.
+Check out the <a href="{{ site.baseurl }}/v1_3_1-docs/installation">Installation</a> page.
 
-## 3. Annotate Content
+## 3. Play
 
-See the <a href="{{ site.baseurl }}/v1_0_0-docs/annotating_content">Annotating Content</a> page.
-
-## 4.
-
-## 5. Take a break
-
-Now that you have setup the concept extraction components successfully, you deserve a break... Find out how in the HowTo section.
+Now that you have setup the concept extraction components successfully, you can start playing with the service. Check out the <a href="{{ site.baseurl }}/v1_3_1-docs/annotating_content">Annotating Content</a> page to
+ start annotating.
```

Comment:
In order to see the whole text in the IDE without scrolling. I think it's fine.
<img width="1404" alt="screen shot 2015-07-07 at 4 47 01 pm" src="https://cloud.githubusercontent.com/assets/3405260/8547476/d67b97fc-24c7-11e5-9588-538e34c97671.png">